rinja_derive/src/config.rs

```diff
@@ -360,6 +360,26 @@ impl<'a> RawSyntax<'a> {
             }
         }
 
+        for end in [syntax.block_end, syntax.expr_end, syntax.comment_end] {
+            for prefix in ["<<", ">>", "&&", "..", "||"] {
+                if end.starts_with(prefix) {
+                    let msg = if end == prefix {
+                        format!("a closing delimiter must not start with an operator: {end:?}")
+                    } else {
+                        format!(
+                            "a closing delimiter must not start an with operator: \
+                             {end:?} starts with {prefix:?}",
```

Comment:
Same here with a little tweak:

```
a closing delimiter must not start with an operator. Expression block closing delimiter ({end:?}) starts with {prefix:?}
```

Reply:
Makes sense! I changed the text in the other delimiter tests, too.